DB/src/main/java/io/deephaven/db/v2/utils/UpdatePerformanceTracker.java

```diff
@@ -190,12 +192,11 @@ private void finishInterval(final long intervalStartTimeMillis, final long inter
                 encounteredErrorLoggingToMemory =
                         logToMemory(intervalLevelDetails, entry, encounteredErrorLoggingToMemory);
             } else if (entry.intervalInvocationCount > 0) {
-                if (entry.totalUsedMemory > aggregatedSmallUpdatesEntry.totalUsedMemory) {
-                    aggregatedSmallUpdatesEntry.totalUsedMemory = entry.totalUsedMemory;
+                if (entry.startTimeNanos > maxStartTimeNanos) {
```

Comment:
Related to the above, I do not think I correctly understand this code.

Reply:
Look again now, fixed to original intention (made a mistake the first time around).